activerecord/lib/active_record/connection_adapters/abstract/connection_pool.rb

```diff
@@ -864,7 +864,7 @@ def connection_pool_list
       alias :connection_pools :connection_pool_list
 
       def establish_connection(config)
-        resolver = ConnectionSpecification::Resolver.new(Base.configurations)
+        resolver = ConnectionSpecification::Resolver.new(Base.local_configurations)
```

Comment:
👍 agree with Kasper, the naming here is difficult to understand; it took me several read throughs to understand the difference between `local_configurations` and `configurations` 😬